src/main/webapp/Validation/sample-clazz.jelly

```diff
@@ -0,0 +1,3 @@
+<f:entry title="${%Positive integer}" field="name">
```

Comment:
Double checking: Should this be `Non-negative integer` or `Positive integer`?

Reply:
Thanks, I first had the minimum 1 and changed it later again to zero, because I think this is more commonly used.
And then forgot to rephrase the title!

Reply:
Cool, no worries!